app/controllers/admin/worldwide_offices_controller.rb

```diff
@@ -23,7 +23,7 @@ def update
 
     if (contact_numbers_attributes = update_params.dig(:contact_attributes, :contact_numbers_attributes))
       contact_numbers_attributes.each do |_id, contact_number_attribute|
-        contact_number_attribute[:_destroy] = true if contact_number_attribute[:label].empty? && contact_number_attribute[:number].empty?
+        contact_number_attribute[:_destroy] = true unless contact_number_attribute[:label].present? && contact_number_attribute[:number].present?
```

Comment:
Will this mark for destruction if any one of either the number and label are not present?

Two tests with one for each of `label` and `number` being blank would clarify this.

Reply:
Updated, can confirm it will mark a contact number for destruction if it's missing either field.